doc/releases/changelog-dev.md

```diff
@@ -69,6 +69,10 @@
 
   * :class:`~.OutPoly`
 
+* A new function called :func:`~.math.choi_matrix` is available, which computes the [Choi matrix](https://en.wikipedia.org/wiki/Choi%E2%80%93Jamio%C5%82kowski_isomorphism) of a quantum channel.
+  This is a useful tool in quantum information science and to check circuit identities involving non-unitary operations.
+  [(#7951)](https://github.com/PennyLaneAI/pennylane/pull/7951)
+
```

Comment:
````suggestion

  ```pycon
  >>> import numpy as np
  >>> Ks = [np.sqrt(0.3) * qml.CNOT((0, 1)), np.sqrt(1-0.3) * qml.X(0)]
  >>> Ks = [qml.matrix(op, wire_order=range(2)) for op in Ks]
  >>> Lambda = qml.math.choi_matrix(Ks)
  >>> np.trace(Lambda), np.trace(Lambda @ Lambda)
  (np.float64(1.0), np.float64(0.58))
  ```

````